rescript/merge.py

```diff
@@ -71,6 +71,9 @@ def merge_taxa(data: pd.DataFrame,
     else:
         result['Taxon'] = result['Taxon'].apply(lambda x: ';'.join(x))
 
+    # fill unassigned taxa, if any
+    result['Taxon'].replace('', 'Unassigned', inplace=True)
```

Comment:
I think we should default to 'Unassigned' but this label should be exposed, so that users can make it consistent with whatever their taxonomy is (e.g., if importing an unusual taxonomy or using taxonomy classifications from something other than q2-feature-classifier). In 99% of cases in the wild 'Unassigned' is fine, but might as well expose. 🤷‍♂️

Reply:
Gotcha, will add!